pipelines/main/platforms/test_macos.yml

```diff
@@ -0,0 +1,64 @@
+steps:
+  - group: ":macos: Macos"
+    steps:
+      - label: ":macos: :test_tube: test ${TRIPLET?}"
+        key: "test_${TRIPLET?}"
+        depends_on:
+          - "build_${TRIPLET?}"
+        plugins:
+          - JuliaCI/external-buildkite#v1:
+              version: ".buildkite-external-version"
+              repo_url: "https://github.com/JuliaCI/julia-buildkite"
+        env:
+          JULIA_SHELL: "/bin/bash"
+        timeout_in_minutes: ${TIMEOUT?}
+        soft_fail: ${ALLOW_FAIL?}
+        commands: |
+          # First, get things like `LONG_COMMIT` and `SHORT_COMMIT`, etc...
+          TRIPLET="${TRIPLET?}" source .buildkite/utilities/calc_version_envs.sh
+
+          echo "--- Download build artifacts"
+          buildkite-agent artifact download "$${UPLOAD_FILENAME}" .
+
+          echo "--- Extract build artifacts"
+          tar xzf "$${UPLOAD_FILENAME}" "$${JULIA_INSTALL_DIR}/"
+
+          echo "--- Ad-hoc sign for testing"
+          contrib/codesign.sh "-" "$${JULIA_INSTALL_DIR}"
+
+          echo "--- Print Julia version info"
+          $${JULIA_BINARY} -e 'using InteractiveUtils; InteractiveUtils.versioninfo()'
+          echo "JULIA_CPU_THREADS is: $${JULIA_CPU_THREADS}"
+          $${JULIA_BINARY} -e '@info "" Sys.CPU_THREADS'
+
+          echo "--- Set some environment variables"
+          export OPENBLAS_NUM_THREADS=4
```

Comment:
Hmmm, I just saw this is in Linux as well.  Why?  @DilumAluthge

Reply:
Which line? The `OPENBLAS_NUM_THREADS`?